### PR TITLE
Changed position of notifications

### PR DESCRIPTION
**Pull request by DaniilPoiarkov**

### CODE REVIEW

**Review by andpii — DISMISSED** [review]

**Review by Vadis00 — DISMISSED** [review]

File: frontend/src/app/components/backlog/sprint/issue/issue.component.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ export class IssueComponent implements OnInit {
     public userServise: UserService,
     public taskServise: TaskService,
     public sprintService: SprintService,
-    public toastrService: ToastrService,
+    public toastrService: NotificationService,
```
Comment (andpii) [inline frontend/src/app/components/backlog/sprint/issue/issue.component.ts]: Please rename name of the service too

File: frontend/src/shared/components/select-users/select-users.component.ts [hunk 2]
```diff
@@ -56,7 +56,7 @@ export class SelectUsersComponent implements OnInit {
   @Output() onDelete = new EventEmitter<string>();
   @Output() onUpdate = new EventEmitter<IUserCard>();
 
-  constructor(private service: BoardService, private toastr: ToastrService) {
+  constructor(private service: BoardService, private toastr: NotificationService) {
```
Comment (andpii) [inline frontend/src/shared/components/select-users/select-users.component.ts:59]: And here too

File: frontend/src/app/components/backlog/sprint/sprint.component.ts [hunk 3]
```diff
@@ -81,7 +81,7 @@ export class SprintComponent implements OnInit, OnChanges {
   constructor(
     public sprintService: SprintService,
     public taskService: TaskService,
-    public toastrService: ToastrService,
+    public toastrService: NotificationService,
```
Comment (andpii) [inline frontend/src/app/components/backlog/sprint/sprint.component.ts]: And here

File: frontend/src/app/components/backlog/backlog.component.ts [hunk 4]
```diff
@@ -69,7 +69,7 @@ export class BacklogComponent implements OnInit {
     public projectService: ProjectService,
     public sprintService: SprintService,
     public currentUserService: GetCurrentUserService,
-    private toastrService: ToastrService,
+    private toastrService: NotificationService,
```
Comment (andpii) [inline frontend/src/app/components/backlog/backlog.component.ts]: Please rename name of the service too

**Review by Vadis00 — APPROVED** [review]

**Review by andpii — APPROVED** [review]

**Pull request merged**